.github/workflows/docs.yml

```diff
@@ -7,6 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     env:
       RUST_TOOLCHAIN: "nightly-2022-08-05"
+      RUSTDOCFLAGS: "-D warnings"
```

Comment:
Not sure if the docs CI job is only run when a commit is successfully merged. If so, a commit can be merged, and then the doc job fails. Not sure if this is the expected behavior we want. Can you confirm this @gpmayorga ? I would expect that a commit with bad documentation can not be merged, but this would increase the CI time for each PR.